dist/404.html

```diff
@@ -22,23 +25,22 @@
             body {
                 display: table-cell;
                 vertical-align: middle;
-                margin: 2em auto;
             }
 
             h1 {
                 color: #555;
                 font-size: 2em;
                 font-weight: 400;
+                margin-top: -2em;
             }
 
             p {
                 margin: 0 auto;
-                width: 280px;
+                width: 30%;
             }
 
             @media only screen and (max-width: 280px) {
```

Comment:
This should be at least 480px IMO.

It looks like this on a smaller screen currently, which I don't think is nice/correct.:

![grafik](https://user-images.githubusercontent.com/21335202/69857838-04e69680-1291-11ea-8053-02ff041bb6b8.png)

Reply:
Hm - this should improve things to a similar state than before:
```suggestion
                width: 280px;
                min-width: 280px;
```
Please test, let me know.

Reply:
Oh no, something weird going on here

Reply:
Looks definitely better!